includes/Database_Upgrader.php

```diff
@@ -58,6 +58,7 @@ public function init() {
 		$routines = [
 			'1.0.0' => 'upgrade_1',
 			'2.0.0' => 'v_2_replace_conic_style_presets',
+			'2.0.1' => 'v_2_add_term',
```

Comment:
According to [semver](https://semver.org/) it should be `2.1.0`?

Reply:
Hmm 🤔 gonna be tricky changing that now after merging

Reply:
No real world impact here, just wanted to be clear here since I was not sure about my own SemVer change.

Reply:
Nobody is using it like a regular package, so my bug fix with 2.1.1 will not break usages that will just accept bug fixes.